ansible_base/jwt_consumer/common/auth.py

```diff
@@ -56,6 +56,8 @@ def parse_jwt_token(self, request):
             },
         )
 
+        setattr(user, "resource_api_actions", validated_body["resource_api_actions"])
```

Comment:
I don't know where this comes from or what content it might have, but I filed AAP-19984 a while ago. Have a look at that and let me know if it's hitting somewhere in the neighborhood of your plan.